config/webpack/webpack.config.js

```diff
@@ -222,7 +222,17 @@ const buildWebpackConfigs = builds.map(({ name, paths, env }) => {
       },
       plugins: [
         new webpack.DefinePlugin(envVars),
-        new StaticSiteGeneratorPlugin()
+        ...locales.map(
```

Comment:
The documentation says it will only render a single file in dev mode, but this appears to always map over every locale?